master/dashboard/views.py

```
@@ -160,150 +235,221 @@ def submit_inference(request):
+                logger.error(f'Inference request {request_id} failed: {error_message}')
+    except Exception as e:
+        logger.critical(f'Fatal error processing inference request {request_id}: {str(e)}')
+        try:
+            # Try to update the request status if possible
+            InferenceRequest.objects.filter(id=request_id).update(
+                status='failed',
+                error=f'Fatal error: {str(e)}',
+                completed_at=timezone.now()
+            )
+        except:
+            pass  # If we can't update the database, we've already logged the error
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)